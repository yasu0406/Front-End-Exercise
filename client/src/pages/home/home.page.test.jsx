import React from 'react';
import { shallow } from 'enzyme';
import Home from './home.page';

it('should render Home component', () => {
    expect(shallow(<Home />)).toMatchSnapshot();
});