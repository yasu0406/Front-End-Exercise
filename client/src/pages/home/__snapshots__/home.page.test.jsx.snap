// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home component 1`] = `ShallowWrapper {}`;
